src/services/dataService.js

```diff
@@ -602,13 +602,17 @@ DataService.prototype.createStream = function(resource, name, context, opts, isR
 //            pollInterval: in milliseconds, how long to wait between polling the server
 //                    only applies if poll=true.  Default is 5000.
 //            http:   similar to .get, etc. at this point, only used to pass http.params for filtering
+//            skipDigest: will skip the $apply & avoid triggering a digest loop
+//                    if set to `true`.  Is intentionally the inverse of the invokeApply
+//                    arg passed to $timeout (due to default values).
 //            errorNotification: will popup an error notification if the API request fails (default true)
 // returns handle to the watch, needed to unwatch e.g.
 //        var handle = DataService.watch(resource,context,callback[,opts])
 //        DataService.unwatch(handle)
   DataService.prototype.watch = function(resource, context, callback, opts) {
     resource = APIService.toResourceGroupVersion(resource);
     opts = opts || {};
+    var invokeApply =  !_.get(opts, 'skipDigest');
```

Comment:
Haha, it can... oi, my first pass was convoluted :smile:  
Was overthinking it, worried I'd accidentally wipe out the default `true` for `$timeout`.

Updated.  This does work, I have tested the following on the pods page:

```javascript 
watches.push(DataService.watch("pods", context, function(pods) {
       // do things that won't update UI
}, { skipDigest: true }));
```
And am manually deploying pods from other pages & can verify no updates.